### ISSUE TITLE
URL 无效不可访问

### ISSUE THREAD

**Issue by kylezhang** [OCselected/ttoos#248]
下面摘出来的内容，对应的 URL 不可访问，多为 `404`

- [`OCSelected` ](https://github.com/OCselected/ttoos#%E4%B8%BA%E4%BB%80%E4%B9%88%E4%BC%9A%E6%94%BE%E5%9C%A8ocselected%E9%A1%B9%E7%9B%AE%E4%B8%8B)
- [`开源之道阅读指南`](https://github.com/OCselected/ttoos/blob/master/content/Community/CONTRIBUTING.md#%E5%89%8D%E6%8F%90)
- [`开源之道文章写作风格指南`、`开源之道需要你的帮助`](https://github.com/OCselected/ttoos/blob/master/content/Community/CONTRIBUTING.md#%E5%89%8D%E6%8F%90)

**Comment by lijiangsheng1** [comment]
all of broke link are in  [CONTRIBUTING.md](https://github.com/OCselected/ttoos/blob/master/content/Community/CONTRIBUTING.md) ?

**Comment by kylezhang** [comment]
> all of broke link are in [CONTRIBUTING.md](https://github.com/OCselected/ttoos/blob/master/content/Community/CONTRIBUTING.md) ?

Yes, the invalid links are from the [CONTRIBUTING.md](https://github.com/OCselected/ttoos/blob/master/content/Community/CONTRIBUTING.md) 。

**Comment by tisonkun** [comment]
@lijiangsheng1 verify the issue. What is the correct link or we don't maintain them any more?
